webdevbroz/pomodoro-app
# Add formatting tools

## Goal
This PR upgrades dependencies in order to be able to apply a combination of ESLint and Prettier to format all the JavaScript code base using standard setup as much as possible.

For now, the files have not been formatted in this PR because we want to format all the code base in a separate PR to avoid mixing manual changes and massive automatic formatting.

#### Added packages
- @ianvs/prettier-plugin-sort-imports
- @typescript-eslint/eslint-plugin
- eslint-config-prettier
- prettier
- prettier-plugin-tailwindcss
- eslint-plugin-tailwindcss

## How to test
Check the following commands work by correctly applying the prettier and lint rules:

- "lint": "next lint"
- "lint:fix": "next lint --fix"
- "format": "prettier --check --ignore-path .gitignore ."
- "format:fix": "prettier --write --ignore-path .gitignore ."

## Screenshots
N/A
